src/domain/Car.js

```diff
@@ -0,0 +1,38 @@
+import { MOVE_SETTING } from '../util/Constant.js';
+
+class Car {
+  #name;
+  #position = [];
+
+  constructor(carName) {
+    this.#name = carName;
+  }
+
+  move(randomNumbers) {
+    randomNumbers.forEach((number) => {
+      this.#position.push(
+        number >= MOVE_SETTING.advanceBoundary
+          ? MOVE_SETTING.advance
+          : MOVE_SETTING.stop
+      );
```

Comment:
number가 MOVE_SETTING.advanceBoundary보다 같거나 큰 조건은 어떤 상황을 의미하는 것일까요? 아래와 같이 변수의 이름으로 추가로 네이밍해줄 수 있을 것 같습니다.

```suggestion
      const isMovable = number >= MOVE_SETTING.advanceBoundary;
      this.#position.push(isMovable ? MOVE_SETTING.advance : MOVE_SETTING.stop);
```

Reply:
좋은 의견 감사합니다!